src/test/java/com/mikhail/tarasevich/charcounter/validator/ValidatorImplTest.java

```diff
@@ -0,0 +1,39 @@
+package com.mikhail.tarasevich.charcounter.validator;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import static org.assertj.core.api.Assertions.assertThatCode;
+
+class ValidatorImplTest {
+
+	ValidatorImpl validatorTest = new ValidatorImpl();
+
+	@Test
+	void validate_InputWhiteSpaces_ExpectedIllegalArgumentException() {
+		Assertions.assertThrows(IllegalArgumentException.class, () -> {
```

Comment:
please use static import here and below
and we use assertj librarty for test, here you use jupiter.api.Assertions
same about other

Reply:
rewrote for assertj.